src/prefect_server/graphql/schema/agents.graphql

```diff
@@ -0,0 +1,76 @@
+extend type Mutation {
+  # ------------------------------------------------------
+  # agents
+  # ------------------------------------------------------
+
+  "Register an agent"
+  register_agent(input: register_agent_input!): agent_id_payload
+
+  "Delete an agent"
+  delete_agent(input: delete_agent_input!): success_payload
+
+  "Create a new agent config"
+  create_agent_config(input: create_agent_config_input!): agent_config_id_payload
+
+  "Delete an agent config"
+  delete_agent_config(input: delete_agent_config_input!): success_payload
+
+  "Update an agent config's config"
+  set_agent_config(input: set_agent_config_input!): success_payload
```

Comment:
If we have multiple atomic mutations (`set_agent_config_settings`, `set_agent_config_name`, `set_agent_config_notification_preferences`) then I like "set" here; if this will eventually become one large mutation I prefer "update" instead: `update_agent_config`